legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/api/BaseGitLabApi.java

```diff
@@ -240,18 +265,45 @@ protected static String getWorkspaceBranchNamePrefix(WorkspaceType workspaceType
         }
     }
 
-    protected static WorkspaceInfo parseWorkspaceBranchName(String branchName)
+    protected static WorkspaceInfo parseWorkspaceBranchName(String branchName, String patchReleaseVersion)
```

Comment:
We shouldn't need anything other than the branch name for parsing. All needed information should be in the branch name itself.